source/applications/advanced/gamma_correction.py

```diff
@@ -0,0 +1,100 @@
+"""
+Capture a 2D image and apply a configurable gamma correction.
+"""
+
+import argparse
+from pathlib import Path
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    """Configure argparse.
+
+    Returns:
+        argparser arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        description="Capture 2D image and apply gamma correction\
+            \nExample:\n\t $ python gamma_correction.py 2",
```

Comment:
If you want to do string concatenation you can do it like this (Note: no comma):
```
(
    "Capture 2D image and apply gamma correction\n"
    "Example:\n\t $ python gamma_correction.py 2"
)
```

Reply:
I'm modifying according to your comment. However, why was it wrong it first place? None of our linters caught it. Is it because you don't like to place `\n` before a line?

Reply:
I reacted to using a `\` to extend a line in python. I think the code is easier to read if we split the string, but not the logic.
I feel `\` is more of a legacy command, like you would using when creating `bash` scripts
The `\n` are fine, it is just the `\` that I have a problem with